src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -677,6 +680,15 @@ private String getCommitHash(Map<String, Object> buildMap, SubmissionType submis
         return result.getFeedbacks();
     }
 
+    private String processResultErrorMessage(final ProgrammingLanguage programmingLanguage, final String message) {
+        if (programmingLanguage == ProgrammingLanguage.JAVA) {
+            return message.split("\\n", 2)[0]
```

Comment:
```suggestion
            // Splitting string at the first linebreak to only get the first line of the Exception
            return message.split("\\n", 2)[0]
```
We should keep the comment here.

Reply:
Added the comment again